src/tls13.c

```diff
@@ -3173,6 +3173,9 @@ static int DoPreSharedKeys(WOLFSSL* ssl, const byte* input, word32 helloSz,
         break;
     }
 
+    if (current == NULL)
+        return 0;
```

Comment:
Is returning 0 correct here?  And then skipping the hash of the rest of client hello for example?